src/pharmpy/tools/amd/run.py

```diff
@@ -270,6 +270,12 @@ def run_amd(
                 f"Unrecognized section {section} in order. Must be one of {default_order}"
             )
 
+    # Filter data to only contain dvid=1
```

Comment:
I don't understand this comment. Where does the filtering happen?

Reply:
In the if-statement below the comment. The original dataset is saved and the model with the filtered dataset is outputted from `create_pk_model`. I believe @antoniaih is working on renaming this function.

Reply:
Ok so this is simply keeping the original dataset to be able to put it back later.

Reply:
If all underlying functions are behaving there should be no need for the defensive copy.

Reply:
Yes precisely. If you are referring to `.copy()` I can remove it. It was mainly added as a safety :)